test/acceptance/utils/StubForWireMock.scala

```diff
@@ -38,6 +47,39 @@ trait StubForWireMock {
   def authenticate() = {
     stubFor(post(urlEqualTo(authUrl))
       .withRequestBody(equalToJson(authRequestJson()))
+      .willReturn(
+        aResponse()
+          .withStatus(Status.OK)
+          .withBody(
+            s"""{"allEnrolments": [{
+               | "key": "HMRC-CUS-ORG",
+               | "identifiers": [
```

Comment:
you might not need to pass any identifiers here, as the implementation is checking only `internalId`. You could have ```identifiers: []``